dtcInterfaceLib/CMakeLists.txt

```diff
@@ -38,3 +40,19 @@ install_source()
 cet_script( ALWAYS_COPY updateMu2e.sh Setup_DTC.sh JAConfig.sh)
 
 add_subdirectory(test)
+
+# Add subdirectories for each language if desired
+
+message(EMOE-001!! BUILD_PYTHON_INTERFACE=$ENV{BUILD_PYTHON_INTERFACE})
```

Comment:
I'm not sure about the messages here. I'd either remove them or make them more self-explanitory

Reply:
deleting is the way to go, it is a debugging printout

Reply:
done

Reply:
just don't delete branch=pasha